cli/config/configuration-qemu.toml.in

```diff
@@ -219,6 +219,11 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Default false
 #enable_template = true
 
+# Specifies the path of template.
+#
+# Default "/run/vc/vm/template"
+#template_path = "/run/vc/vm/template"
```

Comment:
Do you also need to update `configuration-fc.toml.in`?

Reply:
> Do you also need to update `configuration-fc.toml.in`?

@jodh-intel fc cannot support template now.